metadata-ingestion/docs/sources/bigquery/bigquery_pre.md

```diff
@@ -85,6 +85,10 @@ If you have multiple projects in your BigQuery setup, the role should be granted
      client_id: "123456678890"
    ```
 
+##### Profiling Requirements
+
+- To profile BigQuery external tables backed by google drive, grant `Viewer` access to service account's email address (`client_email` in credentials above).
```

Comment:
It would also be preferable to have some... Screenshot :)

Reply:
added screenshot, elaborated content.

Reply:
@mayurinehate Images are put in this repository https://github.com/datahub-project/static-assets to avoid the main repository's size from increasing.

Reply:
Oh! I'll create a separate PR to add image here - https://github.com/datahub-project/static-assets/tree/main/imgs/integrations/bigquery then .

Reply:
done. PR on static-assets repo - https://github.com/datahub-project/static-assets/pull/42